src/generate-note-controller.js

```diff
@@ -223,7 +223,7 @@ module.exports = function(scribe){
         if(markers.length >= 0){
           //check that the selection is within a note
           config.get('selectors').forEach((selector)=>{
-            if(isSelectionEntirelyWithinNote(markers, selector.tagName)){
+            if(isSelectionWithinNote(markers, selector.tagName)){
```

Comment:
It does solve this issue, and all tests pass. But thinking about it maybe you're right. To check that it's entirely within might make more sense.